### ISSUE TITLE
Swap completed but UI shows failed

### ISSUE THREAD

**Issue by jorisvial** [KomodoPlatform/atomicDEX#22]
Full log can be found here https://pastebin.com/m5rwQmsc

```      {
         "event" : {
            "type" : "Finished"
         },
         "timestamp" : 1563405903389
```

Transactions are confirmed on chain:
https://explorer.veruscoin.io/tx/8ac77b999ac34d8bccb7f98f2fa847c6bbee073b29759c22240ae5669b88c89b
https://kmdexplorer.io/tx/440b21bc90f9eabb6a5876a1d82a4b4e182dc9305c8b2835023f69899729860d

The application shows the swap failed.

![Screenshot_20190718-083531](https://user-images.githubusercontent.com/9780671/61419732-7ead6180-a93a-11e9-9381-11880de75b16.jpg)
![Screenshot_20190718-083558](https://user-images.githubusercontent.com/9780671/61419733-7f45f800-a93a-11e9-969b-c0855a571940.jpg)

**Comment by jorisvial** [comment]
Could be related to https://github.com/KomodoPlatform/atomicDEX-API/commit/fef7e1d8bca142f72efe3497cfb0c2c149a62ce7 according to user cipi#4502 on Discord.

**Comment by tonymorony** [comment]
It should be fixed in 0.1.3-a and I'm not able to reproduce at the moment
